content/en/security_platform/cloud_siem/signal_correlation_rules.md

```diff
@@ -0,0 +1,156 @@
+---
+title: Signal Correlation Rules
+type: documentation
+further_reading:
+- link: "/cloud_siem/explorer/"
+  tag: "Documentation"
+  text: "Learn about the Security Signals Explorer"
+- link: "/security_platform/notifications/variables/"
+  tag: "Documentation"
+  text: "Learn more about Security notification variables"
+---
+
+## Overview
+
+Signal correlation rules combine multiple signals together to generate a new signal so that you can alert on more complex use cases and reduce alert fatigue. For example, you can correlate events or signals to identify a specific issue or generate an alert only if a specific `low` severity signal is combined with a specific `high` severity signal.
+
+For example, you can create a signal by combining these two rules:
+
+1. Detect if an access attempt was made from an expired account
+2. Detect if there was an attempt to authenticate into a host or resource
+
+And use the `expired account ID` attribute to correlate the two rules. 
+
+You can correlate log detection and signal correlation rules, as well as log detection rules with Cloud Workload Security and Application Security Monitoring rules.
```

Comment:
In fact, it is not possible to correlate a `log detection` rule with a `signal correlation` rule. To be more generic, it is not possible to use a `signal correlation` rule inside a new `signal correlation` rule. 
We decided to not support this use case to avoid infinite recursion.
On the other hand, the other correlation are fine.

Reply:
Thanks, updated and removed the `signal correlation` part.